### PR TITLE
initial commit

### PR DESCRIPTION
**Pull request by Phmank**

### CODE REVIEW

File: frontend/components/AddToCartBar/index.jsx [hunk 1]
```diff
@@ -0,0 +1,213 @@
...
+
+/**
+ * AddToCartBar component
+ * @param {Object} props Component props
+ * @returns {JSX}
+ */
+const AddToCartBar = ({
+  handleAddToCart, resetClicked, loading, disabled, conditioner,
+}) => {
+  const { setQuantity: setContextQuantity, productId } = useCurrentProduct();
```
Comment (patrick-scherer) [inline frontend/components/AddToCartBar/index.jsx]: `productId` is not used + some eslint errors

File: frontend/components/AddToCartBar/index.jsx [hunk 2]
```diff
@@ -0,0 +1,213 @@
...
+      // Update the product context
+      setContextQuantity(parseInt(inputQuantity, 10));
+      // Trigger addToCart
+      handleAddToCart();
+    });
+  }), [conditioner, handleAddToCart, inputQuantity, setContextQuantity]);
+
+  const handleOnKeyPress = useCallback((e) => {
+    // Simulate button click when enter key was pressed
+    if (e.nativeEvent.keyCode === 13) {
```
Comment (patrick-scherer) [inline frontend/components/AddToCartBar/index.jsx]: Enter key does not work - remove code? but the Enter key works in the cart

Comment (Phmank) [reply frontend/components/AddToCartBar/index.jsx]: removed

File: frontend/components/AddToCartBar/index.jsx (hunk 1 above)
Comment (Phmank) [reply frontend/components/AddToCartBar/index.jsx]: removed

**Pull request merged**